### ISSUE TITLE
Crash when opening NextSeq screen

### ISSUE THREAD

**Issue by izzyreal** [izzyreal/mpc#31]
Reported by Eagle Andrii‎
```
Thread 0 Crashed:: JUCE Message Thread Dispatch queue: com.apple.main-thread
0 nl.izmar.vmpc 0x000000010826d6a4 moduru::observer::Observable::deleteObservers() + 4
1 nl.izmar.vmpc 0x00000001084f82e7 mpc::ui::sequencer::NextSeqPadObserver::NextSeqPadObserver(mpc::Mpc*) + 631
2 nl.izmar.vmpc 0x000000010840a0d2 mpc::lcdgui::LayeredScreen::initObserver() + 1282
3 nl.izmar.vmpc 0x00000001084095a4 mpc::lcdgui::LayeredScreen::openScreen(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >) + 3012
4 nl.izmar.vmpc 0x000000010838999d mpc::controls::sequencer::NextSeqControls::function(int) + 125
5 nl.izmar.vmpc 0x00000001083f18fe mpc::hardware::Button::push() + 1598
6 nl.izmar.vmpc 0x0000000108559699 ButtonControl::mouseDown(juce::MouseEvent const&) + 57
```

**Comment by izzyreal** [comment]
Opening the screen (now?) works, but opening the PAD view makes it crash.

**Comment by izzyreal** [comment]
Resolved in 408ba5ac300acad0f283bfc9c058b78568bc84ea
